apps/frontend/src/app/3_organisms/Header/components/ProductLinks/ProductLinks.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { FC, useCallback, useRef, useState } from 'react';
+
+import classNames from 'classnames';
+import { t } from 'i18next';
+
+import { useOnClickOutside } from '@sovryn/ui';
+
+import { ReactComponent as AppsIcon } from '../../../../../assets/images/apps-icon.svg';
+import { ALPHA_LINKS } from '../../../../../constants/links';
+import { translations } from '../../../../../locales/i18n';
+import { Environments } from '../../../../../types/global';
+import { isMainnet, isStaging } from '../../../../../utils/helpers';
+import { ProductLink } from '../ProductLink/ProductLink';
+import styles from './ProductLinks.module.css';
+
+const linkToAlpha = (path?: string) =>
```

Comment:
Extract this logic (without path concatenation) to `helpers.ts` (similar to `D2_URL` in D1).